vmod/vmod_directors_fall_back.c

```diff
@@ -175,6 +175,16 @@ vmod_fallback_resolve(VRT_CTX, VCL_BACKEND dir)
 	return (be);
 }
 
+static void v_matchproto_(vdi_destroy_f)
+vmod_fallback_release(VCL_BACKEND dir)
```

Comment:
Wrong proto `vdi_destroy_f` everywhere except for the shard director.

Reply:
thx

Reply:
force pushed a fix